CHANGELOG.md

```diff
@@ -460,7 +460,7 @@ from now on the only officially supported cairo compiler version is 2
 
 ### Changed
 
-- `--add-profile` no longer accepts `-a` for short
+- `--update-profile` no longer accepts `-a` for short
```

Comment:
```suggestion
- `--add-profile` no longer accepts `-a` for short
```
let's not update older changelog messages

Reply:
This has been updated in the new commit